src/signal-server.cc

```diff
@@ -939,22 +929,13 @@ void ObstructionAnalysis(struct site xmtr, struct site rcvr, double f, FILE *out
 
 void resize_elev(struct output &out) { out.elev.resize(ARRAYSIZE + 10, 0.0); }
 
-void resize_path(struct path &path)
-{
-    path.length = 0;
-    path.lat.resize(ARRAYSIZE, 0.0);
-    path.lon.resize(ARRAYSIZE, 0.0);
-    path.elevation.resize(ARRAYSIZE, 0.0);
-    path.distance.resize(ARRAYSIZE, 0.0);
-}
-
 int init(const char *sdf_path, bool debug)
 {
     // these can stay globals
     G_gpsav = 0;
     G_sdf_path[0] = 0;
     G_fzone_clearance = 0.6;
-    G_earthradius = EARTHRADIUS;
+    G_earthradius_ft = EARTHRADIUS;
```

Comment:
Should this be EARTHRADIUS_ft? why does it even copy one constant into another?

Reply:
I wondered the same thing. Gonna remove that global.